docs/diagnostics/PX1007.md

```diff
@@ -5,18 +5,26 @@ This document describes the PX1007 diagnostic.
 
 | Code   | Short Description                                   | Type                             | Code Fix  | 
 | ------ | ----------------------------------------------------| -------------------------------- | --------- | 
-| PX1007 | A DAC, a DAC extension, and public DAC properties should have a description in the `summary` XML tag. | Warning | Available | 
+| PX1007 | The DAC, DAC extension, or DAC property should have a description in the "summary" XML tag or "inheritdoc" XML tag | Warning | Available | 
 
 ## Diagnostic Description
 Any DAC or DAC extension and their public property fields should have one of the following:
 * A description in the `summary` XML tag
+* One or multiple `inheritdoc` tags in any format on DACs, DAC extensions, and DAC field properties of non-projection DACs (including unmapped properties of projection DACs)
 * The 'inheritoc' XML tag on mapped DAC fields of projection DACs
+* The `exclude` tag 
+  For more details, on the `inheritdoc` tag, see the [inheritdoc](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#inheritdoc) tag documentation.
 The descriptions of the DACs, DAC extensions, and their members can be displayed in the DAC Schema Browser and used for building the API and DAC references for the Acumatica ERP libraries.
 
-The code fix does one of the following:
+> **Note**
+> For the mapped DAC fields, the original mapped property is determined from the `BqlField` or `BqlTable` values specified in attributes of the projection DAC property. Non-mapped properties (unbound or calculated at runtime) are processed as normal DAC properties.
 
+The code fix does one of the following:
  - Adds the `summary` XML tags, in which you type the description of the code item
  - Adds the `exclude` XML tag, which excludes the item from the API and DAC References
+ - Adds the `inheritdoc` tag to a mapped DAC property of a projection DAC.
+   To apply the code fix and add the `inheritdoc` tag to multiple DACs, you can use the **Fix All** button in the code fix menu of Visual Studio. Clicking this button will automatically generate correct documentation for most of the projection DAC properties.
+   This code fix also deletes incorrect documentation tags like `summary` or other incorrectly declared `inheritdoc` tags that should not be placed on the mapped property of a projection DAC.
 
 The new diagnostic is available only if the value of **Tools > Options > Acuminator > Code Analysis > Enable PX1007 diagnostic** is _True_. By default, the value of this option is _False_. 
```

Comment:
Maybe we should extend the guide to enable the diagnostic with info from this article but I'm not insisting:
https://wiki.acumatica.com/display/TND/Enabling+PX1007+in+Visual+Studio